src/store/ViewStore.js

```diff
@@ -24,12 +24,13 @@ export default class ViewStore {
     get loader() { return this.mainStore.loader; }
 
     onContextReady = () => {
-        const views = createObjectFromLocalStorage('cq-views');
+        const views = createObjectFromLocalStorage(`${this.chartId}-cq-views`);
```

Comment:
`cq-view` should be shared among all charts, I think. Currently how it is done should stay that way

Reply:
@brucebinary  views will be shared between all instances.